### PR TITLE
Bug 1810535: Independent Dashboard is not showing correct metrics.

### PR DESCRIPTION
**Pull request by bipuladh**
- Changed field name from `enabled` to `enable`
- Updated queries for utilization and breakdown card.
- Update status card to use K8s CR.

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/independent-mode/install.tsx [hunk 1]
```diff
@@ -27,6 +30,10 @@ const ERROR: DataState = {
 
 const getErrorText = (text: string) => <span className="im-install-page--error">{text}</span>;
 
+const mapDispatchToProps = (dispatch: Dispatch): MapDispatchToProps => ({
+  enableDashboard: () => dispatch(setFlag(OCS_INDEPENDENT_FLAG, true)),
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/independent-mode/install.tsx]: So after creating the cluster, we have dashboard enabled ?
If I got right, why this change ?

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-body-independent.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+type BreakdownBodyProps = {
+  isLoading: boolean;
+  hasLoadError: boolean;
+  metricTotal: string;
+  top5MetricsStats: StackDataPoint[];
+  metricModel: K8sKind;
+  humanize: Humanize;
+  ocsVersion?: string;
+};
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-body-independent.tsx]: You can  actually reuse the existing component. You dont need `capacityTotal` for independent mode, and that is already optional for BreakdownBody.
Also just like noobaa, capacity total === metricTotal here.

Refer: https://github.com/openshift/console/blob/f051348ca4db91bbccdf03fde9bc074975348d6d/frontend/packages/noobaa-storage-plugin/src/components/capacity-breakdown/capacity-breakdown-card.tsx#L111-L119

File: frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.ts [hunk 3]
```diff
@@ -27,3 +29,30 @@ export const checkForIndependentSupport = (csv: ClusterServiceVersionKind): bool
...
+  [ClusterPhase.READY]: HealthState.OK,
+  [ClusterPhase.CONNECTING]: HealthState.UPDATING,
+  [ClusterPhase.PROGRESSING]: HealthState.UPDATING,
+  [ClusterPhase.ERROR]: HealthState.ERROR,
+});
+
+export const getClusterHealth = (cluster: K8sResourceKind, loaded: boolean, error): HealthState => {
+  const phase = cluster?.status?.phase;
+  if (!_.isEmpty(error)) {
+    if (error?.response?.status === 404) return HealthState.UNKNOWN;
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.ts]:
```suggestion
    if (error?.response?.status === 404) return HealthState.NOTAVAILABLE;
```

File: frontend/packages/ceph-storage-plugin/src/components/independent-dashboard-page/breakdown-card/card.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        </div>
+      </DashboardCardHeader>
+      <DashboardCardBody classname="ceph-capacity-breakdown-card__body">
+        <BreakdownCardBody
+          isLoading={queriesDataLoaded}
+          hasLoadError={queriesLoadError}
+          metricTotal={metricTotal}
+          top5MetricsStats={top5MetricsStats}
+          metricModel={model}
+          humanize={humanize}
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/independent-dashboard-page/breakdown-card/card.tsx]:
```suggestion
        <BreakdownCardBody
          isLoading={queriesDataLoaded}
          hasLoadError={queriesLoadError}
          metricTotal={metricTotal}
          top5MetricsStats={top5MetricsStats}
          capacityUsed={metricTotal}
          metricModel={model}
          humanize={humanize}
```

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-body-independent.tsx (hunk 2 above)
Comment (bipuladh) [reply frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-body-independent.tsx]: @afreen23 the problem lies with used capacity. I couldn't see a workaround it. Do you have any suggestions around it?

File: frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.ts (hunk 3 above)
Comment (bipuladh) [reply frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.ts]: But OCS Operator is available the storage cluster endpoint is not available,

Comment (afreen23) [reply frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.ts]: I think it will be the status of independent mode cluster. We need not to see wrt operator status, whose status is shown on overview status card

File: frontend/packages/ceph-storage-plugin/src/components/independent-dashboard-page/breakdown-card/card.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    prometheusResults.getIn([queries[q], 'loadError']),
+  );
+
+  const queriesDataLoaded = queryKeys.some((q) => !prometheusResults.getIn([queries[q], 'data']));
+
+  const humanize = humanizeBinaryBytes;
+  const top6MetricsData = getInstantVectorStats(results[0], metric);
+  const top5SortedMetricsData = sortInstantVectorStats(top6MetricsData);
+  const top5MetricsStats = getStackChartStats(top5SortedMetricsData, humanize);
+  const metricTotal = _.get(results[1], 'data.result[0].value[1]');
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/components/independent-dashboard-page/breakdown-card/card.tsx]: can we use optional chaining over here?
